src/admin/paths/admin@statements@upload.yaml

```diff
@@ -0,0 +1,52 @@
+post:
+  tags:
+    - Statements
+  summary: Upload a statement
+  description: |
+    Upload a statement to Fyle for reconciliation.
+  operationId: statements_post
```

Comment:
try to understand what and where this is used

Reply:
Got it, this is a unique string used to identify an operation.
Addl info: Some code generators use this value to name the corresponding methods in code.